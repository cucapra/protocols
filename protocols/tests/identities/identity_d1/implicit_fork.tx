// ARGS: --verilog identities/identity_d1/identity_d1.v --protocol identities/identity_d1/identity_d1.prot
// RETURN: 101
implicit_fork(1, 1);
implicit_fork(2, 2);
implicit_fork(3, 4);
implicit_fork(4, 4);
