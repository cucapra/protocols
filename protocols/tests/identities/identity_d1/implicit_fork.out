error: The two expressions did not evaluate to the same value
   ┌─ identities/identity_d1/identity_d1.prot:12:13
   │
12 │   assert_eq(s, DUT.s);
   │             ^^^^^^^^ LHS Value: ValueOwned(00000000000000000000000000000100), RHS Value: ValueOwned(00000000000000000000000000000011)

